### PR TITLE
[directxtex] Update to dec2019 release

### PR DESCRIPTION
**Pull request by shibayan**
**Describe the pull request**

- Base Issue #9380
- Update version `dec2019`
- Adding build for ARM64 Windows 10
- CI baseline updated `directxtex:arm64-windows`

### CODE REVIEW

**Review by PhoebeHui — APPROVED** [review]

**Comment by PhoebeHui** [comment]
directxtex doesn't contains any features.

File: ports/directxtex/portfile.cmake [hunk 1]
```diff
@@ -55,7 +59,7 @@ file(INSTALL
     ${SOURCE_PATH}/DirectXTex/Bin/${SLN_NAME}/${BUILD_ARCH}/Release/DirectXTex.pdb
     DESTINATION ${CURRENT_PACKAGES_DIR}/lib)
 
-if(NOT VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")
+if(NOT VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore" AND NOT TRIPLET_SYSTEM_ARCH STREQUAL "arm64")
```
Comment (PhoebeHui) [inline ports/directxtex/portfile.cmake]: could you please also replace it with VCPKG_TARGET_IS_UWP?

Comment (shibayan) [reply ports/directxtex/portfile.cmake]: Done!

**Comment by ras0219-msft** [comment]
Thanks for the PR!

**Pull request merged**